docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -17,21 +17,19 @@ anomalies and notifies you in an email. This page helps you to configure an
 == Creating an alert
 
 You can create {anomaly-detect} alerts in the {anomaly-job} wizard after you 
-start the job, from the job list, or under **{stack-manage-app} > 
-{alerts-ui}**. On the *Create alert* window, select *{anomaly-detect-cap} alert* 
-under the {ml-cap} section, then give a name to the alert and optionally provide 
-tags.
+start the job, from the job list, or under **{stack-manage-app} > {alerts-ui}**. 
+On the *Create alert* window, select *{anomaly-detect-cap} alert* under the 
+{ml-cap} section, then give a name to the alert and optionally provide tags.
 
 Specify the time interval for the alert to check detected anomalies. It is 
 recommended to select an interval that is close to the bucket span of the 
 associated job. You can also select a notification option by using the _Notify_ 
-selector. For more details, refer to the documentation of
-{kibana-ref}/defining-alerts.html#defining-alerts-general-details[general alert details].
-
-NOTE: {anomaly-detect-cap} alerts handle duplications. If you set an interval 
-that makes the alert check the same bucket multiple times and the bucket 
-contains an anomaly that meets the alert conditions, the configured action 
-is triggered only once.
+selector. An alert instance remains active as long as anomalies are found for a 
+particular {anomaly-job} during the check interval. When there is no anomaly 
+found in the next interval, the state of the alert instance changes to 
+`Recovered`, then transitions to `OK`. For more details, refer to the
```

Comment:
to alight with the alerting framework terminology, It should be `status`, not `state`. Also, `Recovered` is a built-in action group.

```suggestion
found in the next interval, the `Recovered` action group is invoked and the status of the alert instance changes to `OK`. For more details, refer to the
```